index.html

```diff
@@ -67,11 +66,167 @@ <h3>
         </div>
         <img src="images/Action.svg" alt="action" />
         <div class="my-recent-works">
-          <h3>My Recent</h3>
-          <h3>Works</h3>
+          <h3>My Recent Works</h3>
           <img src="images/Indicator.png" alt="Indicator">
         </div>
       </div>
     </div>
+    <!-- ------------------mobile version----------- -->
+    <div class="my-works">
+      <!-- <h1>My Recent Works</h1> -->
+      <!-- <img src="images/Indicator.svg" alt="Indicator" class="indicator" /> -->
+      <div class="works">
+        <div class="work-1">
+          <div class="img-holder"></div>
+          <div class="posts">
+            <h2>Multi-Post Stories Gain+Glory</h2>
+            <div class="flex-post">
+              <ul>
+                <li><a href="#">Ruby on rails</a></li>
+                <li><a href="#">CSS</a></li>
+                <li><a href="#">Javascript</a></li>
+                <li><a href="#">HTML</a></li>
```

Comment:
- [ ] Good job on adding the list of languages used on this project but kindly remove the a tag from them because they don't need to be linked. They are just the list of languages used in this project. Keep shining 💪